README.md

```diff
@@ -75,19 +75,19 @@ After you're finished please remove all the comments and instructions!
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
 
-<details>
+<!-- <details>
```

Comment:
- [ ] **Good job so far 👏 👏** However, the README is not professional as the **Getting Started** section is repeated which is not clear to the user 😨, I still see the messages made to guide the author of the README how to complete the README with his/her project information eg: `Describe your project in 1 or 2 sentences` or `Describe between 1-3 key features of the application` 😨 Please try to make your README Professional 🙏